### PR TITLE
Feature/tito to events air

### PR DESCRIPTION
**Pull request by JakeGinnivan**

### CODE REVIEW

File: website/app/lib/events-air.server.ts [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+                `https://login.microsoftonline.com/${EVENTS_AIR_TENANT_ID}/oauth2/v2.0/token`,
+                {
+                    method: 'POST',
+                    headers: {
+                        'Content-Type': 'application/x-www-form-urlencoded',
+                    },
+                    body: new URLSearchParams({
+                        client_id: EVENTS_AIR_CLIENT_ID!,
+                        client_secret: EVENTS_AIR_CLIENT_SECRET!,
+                        scope: 'https://eventsairprod.onmicrosoft.com/85d8f626-4e3d-4357-89c6-327d4e6d3d93/.default',
```
Comment (MatthewJNixon) [inline website/app/lib/events-air.server.ts:32]: should this scope be using the EVENTS_AIR_TENANT_ID ? Or the client id? I guess it looks like magic here

Comment (MatthewJNixon) [reply website/app/lib/events-air.server.ts:32]: (85d8f626-4e3d-4357-89c6-327d4e6d3d93 that is)

**Review by MatthewJNixon — APPROVED** [review]
approved with comment for clarity

File: website/app/lib/events-air.server.ts (hunk 1 above)
Comment (JakeGinnivan) [reply website/app/lib/events-air.server.ts:32]: It's a fixed scope

**Pull request merged**